### PR TITLE
Markdown file for localisation testing

### PR DESCRIPTION
**Pull request by RobynCastro**
Needs images.

### CODE REVIEW

File: docs/igvc2018/EKFTesting.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+3. Plug the USB hub into your laptop.
+4. Plug the GPS into your laptop on a separate port from the USB hub.
+5. Place the GPS antenna at the peak of the tower.
+6. Move the test cart to an area with good GPS signal
+
+# Collecting rosbags for testing
+
+1. Open up 4 terminals (or equivalent).
+2. Run 'roscore' on the first terminal.
+2. Run 'cd ~/Snowflake && source devel/setup.sh' on the second terminal.
```
Comment (raadkhan) [inline docs/igvc2018/EKFTesting.md]: should it be ``source devel/setup.bash`` instead?

File: docs/igvc2018/EKFTesting.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+3. Launch the localisation stack by running 'roslaunch localisation_igvc localisation.launch'.
+  - This will launch the EKF, GPS, and IMU.
+4. Check that GPS has fix by running 'rostopic echo /gps/navsatfix' on the third terminal.
+  - If there's an output and the header is constantly changing, then the GPS has a fix.
+  - Make sure the GPS is on it's own port in the laptop to ensure the best fix.
```
Comment (raadkhan) [inline docs/igvc2018/EKFTesting.md]: just curious, what does it mean by own port?

File: docs/igvc2018/EKFTesting.md (hunk 1 above)
Comment (ValRat) [reply docs/igvc2018/EKFTesting.md]: Depends on which shell you use really, I think `.sh` is the most neutral one. I source `setup.zsh` on [zsh](https://en.wikipedia.org/wiki/Z_shell).

File: docs/igvc2018/EKFTesting.md (hunk 2 above)
Comment (ValRat) [reply docs/igvc2018/EKFTesting.md]: USB Port, i.e. not sharing through a hub.

**Comment by ValRat** [comment]
@raadkhan, fixed the ci error [here](https://github.com/UBC-Snowbots/Snowflake/pull/315). Should be good to merge once this gets hotfixed in.

**Pull request merged**